### DISCUSSION TITLE
Is Uno.Gallery considered for Hacktoberfest?

### DISCUSSION THREAD

**Discussion by trungnt2910** [unoplatform/uno#10037] [General]
I see a large list of issues marked as `hacktoberfest` on the [Uno.Gallery](https://github.com/unoplatform/Uno.Gallery) repository, but when I made pull requests for it, this appeared on my dashboard:

![image](https://user-images.githubusercontent.com/57174311/193502952-6a19e183-3973-4482-b1d5-98bd9b3c853b.png)

Apparently, the repo, unlike the main [Uno](https://github.com/unoplatform/uno) repo, does not have the `hacktoberfest` tag.
![image](https://user-images.githubusercontent.com/57174311/193503019-5b0e5aa7-d682-41d8-b350-caaa47c70cb9.png)

Thanks in advance.

**Comment by Youssef1313** [comment]
@jeromelaban Could you take a look please?

**Reply by jeromelaban** [reply]
The tag is now set. Thanks!
